app/src/main/java/com/example/cachua/musicdemo21/data/source/ContentResolverData.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.cachua.musicdemo21.data.source
+
+import android.content.ContentUris
+import android.content.Context
+import android.net.Uri
+import android.provider.MediaStore
+import com.example.cachua.musicdemo21.data.model.MusicModel
+
+class ContentResolverData(private val context: Context) {
+
+    fun getData(): ArrayList<MusicModel> {
+        val musicList: ArrayList<MusicModel> = ArrayList()
```

Comment:
```suggestion
        val musics: ArrayList<MusicModel> = ArrayList()
```
Tương tự với các dãy khác

Reply:
musicList cũng được mà ta